workers/docker-worker/test/integration/docker_manifest_test.js

```diff
@@ -65,4 +65,46 @@ helper.secrets.mockSuite(suiteName(), ['docker', 'ci-creds'], function(mock, ski
     assert.equal(task2.run.state, 'completed', 'task should be successful');
     assert.equal(task2.run.reasonResolved, 'completed', 'task should be successful');
   });
+
+  test('docker manifest v1.2 only', async () => {
+
+    let task1 = await worker.postToQueue({
+      payload: {
+        image: 'tutum/curl',
+        artifacts: {
+          'public/image.tar.zst': {
+            type: 'file',
+            expires: expires(),
+            path: '/image.tar.zst',
+          },
+        },
+        command: [
+          'curl',
+          '-o',
+          '/image.tar.zst',
+          '-L',
+          'https://firefox-ci-tc.services.mozilla.com/api/queue/v1/task/U9nuixDnT_6eGJ-OsKMLCQ/runs/0/artifacts/public/image.tar.zst',
```

Comment:
This probably wants to be copied to https://s3-us-west-2.amazonaws.com/docker-worker-manifest-test/ before landing, since the artifact referenced here will expire in < month.

Reply:
I'll push a change with the new URL.  Can you add some comments as to how the image was created?

Reply:
Oh, I can't push to your repo.  The new URL is `https://s3-us-west-2.amazonaws.com/docker-worker-manifest-test/image-v1_2.tar.zst`.